### PR TITLE
fix(split-pane): show border in RTL mode

### PR DESCRIPTION
**Pull request by ataramez**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [] Tests for the changes have been added (for bug fixes / features)
- [] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [X] Build (`npm run build`) was run locally and any changes were pushed
- [X] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: 20994
[https://github.com/ionic-team/ionic/issues/20994]()

## What is the new behavior?
In split-pane component, RTL mode now shows the border between the two panes

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: core/src/components/split-pane/split-pane.ios.scss [hunk 1]
```diff
@@ -19,6 +24,11 @@
 }
 
 :host(.split-pane-visible) ::slotted(.split-pane-side[side=end]) {
+  @include rtl() {
+    border-right: var(--border);
+    border-left: 0;
+  }
+
```
Comment (liamdebeasi) [inline core/src/components/split-pane/split-pane.ios.scss:31]:
```suggestion
  @include border(0, 0, 0, var(--border));
```

Let's remove the `@include rtl()` stuff and replace it with the mixin.

You will also need to get rid of the `border-left: var(--border);` and `border-right: 0` lines below.

File: core/src/components/split-pane/split-pane.ios.scss [hunk 2]
```diff
@@ -11,6 +11,11 @@
 }
 
 :host(.split-pane-visible) ::slotted(.split-pane-side) {
+  @include rtl() {
+    border-right: 0;
+    border-left: var(--border);
+  }
+
```
Comment (liamdebeasi) [inline core/src/components/split-pane/split-pane.ios.scss:18]:
```suggestion
  @include border(0, var(--border), 0, 0);
```

Let's remove the `@include rtl()` stuff and replace it with the mixin.

You will also need to get rid of the `border-right: var(--border);` and `border-left: 0` lines below.

File: core/src/components/split-pane/split-pane.md.scss [hunk 3]
```diff
@@ -11,6 +11,11 @@
 }
 
 :host(.split-pane-visible) ::slotted(.split-pane-side) {
+  @include rtl() {
+    border-right: 0;
+    border-left: var(--border);
+  }
+
```
Comment (liamdebeasi) [inline core/src/components/split-pane/split-pane.md.scss:18]:
```suggestion
  @include border(0, var(--border), 0, 0);
```

Let's remove the `@include rtl()` stuff and replace it with the mixin.

You will also need to get rid of the `border-right: var(--border);` and `border-left: 0` lines below.

File: core/src/components/split-pane/split-pane.md.scss [hunk 4]
```diff
@@ -19,6 +24,11 @@
 }
 
 :host(.split-pane-visible) ::slotted(.split-pane-side[side=end]) {
+  @include rtl() {
+    border-right: var(--border);
+    border-left: 0;
+  }
+
```
Comment (liamdebeasi) [inline core/src/components/split-pane/split-pane.md.scss:31]:
```suggestion
  @include border(0, 0, 0, var(--border));
```

Let's remove the `@include rtl()` stuff and replace it with the mixin.

You will also need to get rid of the `border-left: var(--border);` and `border-right: 0` lines below.

**Comment by ataramez** [comment]
@liamdebeasi  Yes your are right. But I think you switched the start and end parameters when calling the mixin in your suggested changes. Please check the new commit.

**Comment by liamdebeasi** [comment]
Yes I did! Thanks for catching that 😄

**Comment by liamdebeasi** [comment]
Merged. Thank you!

**Pull request merged**